metal/modules/lstm_module.py

```diff
@@ -84,6 +90,14 @@ def _load_pretrained(self, pretrained):
         embedding.weight.data.copy_(pretrained)
         return embedding
 
+    def _attention(self, output):
```

Comment:
Add a doc string?  Also, if this came from a reference implementation (?) can put explicit link here!